### ISSUE TITLE
Deprecated Images

### ISSUE THREAD

**Issue by johnstcn** [coder/enterprise-images#238]
_TL;DR all images except `enterprise-minimal` and `enterprise-base` are deprecated._

As a number of Docker images in this repository have not been updated in quite some time, we are reducing the scope of this repo to only `enterprise-minimal` and `enterprise-base`.

Specifically, the following images are now deprecated and will no longer be updated.

- [enterprise-android](https://hub.docker.com/r/codercom/enterprise-android)
- [enterprise-clion](https://hub.docker.com/r/codercom/enterprise-clion)
- [enterprise-configure](https://hub.docker.com/r/codercom/enterprise-configure)
- [enterprise-dataspell](https://hub.docker.com/r/codercom/enterprise-dataspell)
- [enterprise-goland](https://hub.docker.com/r/codercom/enterprise-goland)
- [enterprise-golang](https://hub.docker.com/r/codercom/enterprise-golang)
- [enterprise-intellij](https://hub.docker.com/r/codercom/enterprise-intellij)
- [enterprise-java](https://hub.docker.com/r/codercom/enterprise-java)
- [enterprise-jupyter](https://hub.docker.com/r/codercom/enterprise-jupyter)
- [enterprise-multieditor](https://hub.docker.com/r/codercom/enterprise-multieditor)
- [enterprise-node](https://hub.docker.com/r/codercom/enterprise-node)
- [enterprise-pycharm](https://hub.docker.com/r/codercom/enterprise-pycharm)
- [enterprise-ruby](https://hub.docker.com/r/codercom/enterprise-ruby)
- [enterprise-rust](https://hub.docker.com/r/codercom/enterprise-rust)
- [enterprise-vnc](https://hub.docker.com/r/codercom/enterprise-vnc)
- [enterprise-webstorm](https://hub.docker.com/r/codercom/enterprise-webstorm)

The Dockerfiles will remain here for reference.

The below images will continue to be updated:

- [enterprise-minimal](https://hub.docker.com/r/codercom/enterprise-minimal)
- [enterprise-base](https://hub.docker.com/r/codercom/enterprise-base)

If you depend on any of the deprecated images, we would recommend that you build your own custom image to use with Coder.

If you have questions or concerns, please head over to our [Discord Server](https://discord.gg/coder).
